doc/source/diagnostics/slow_extended.rst

```diff
@@ -27,6 +27,39 @@ Other Information
   * Author / Origin:
     Maciej Dobrzanski
 
+Enabling Slow Log Query
```

Comment:
Shall we also describe --slow_query_log parameter which may be passed to mysqld during server startup?

Reply:
Added a note pointing to the MySQL documentation.